tools/prove_wrapper

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# shellcheck disable=SC2048,SC2086
+
+OUTPUT=$(mktemp)
+UNHANDLED=$(mktemp)
+
+echo "Running prove with TAP output check"
+prove "$@" | tee "$OUTPUT"
+STATUS=$?
+
+if [ "$STATUS" -ne 0 ]; then
+    echo "Failed with status $STATUS"
+    exit $STATUS
+fi
+
+grep -vE '^(\[[0-9]{2}:[0-9]{2}:[0-9]{2})' "$OUTPUT" | grep -v '^\s*$' > "$UNHANDLED"
+
+if [ -s "$UNHANDLED" ]; then
+    echo "Unhandled output found:"
+    cat "$UNHANDLED"
+    exit 1
+else
+    echo "Check passed!"
```

Comment:
```suggestion
    echo "No unhandled output found"
```

Or maybe even produce something that is in-line with tap output in all those `echo` calls.

Reply:
What do you mean?

Reply:
Something like:
```suggestion
    echo "ok - no unhandled output found"
```